homework/index.js

```diff
@@ -28,14 +28,75 @@
     });
     return elem;
   }
+  // convert date time
+  function convertDataTime(dateTimeTxt) {
+    const dateTime = new Date(dateTimeTxt);
+    return dateTime.toLocaleString();
+  }
+  // sort repositories by name
+  function sortElements(a, b) {
+    return a.name.localeCompare(b.name);
+  }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const repoList = createAndAppend('li', ul, { class: 'rep-list' });
+    const repositoryName = createAndAppend('p', repoList, {
+      text: ' Repository: ',
+      class: 'repo-details',
+    });
+
+    const description = createAndAppend('p', repoList, {
+      text: ' Description: ',
+      class: 'repo-details',
+    });
+
+    const forkedNumbers = createAndAppend('p', repoList, {
+      text: ' Forks: ',
+      class: 'repo-details',
+    });
+
+    const updatedTime = createAndAppend('p', repoList, {
+      text: ' Updated: ',
+      class: 'repo-details',
+    });
+
+    createAndAppend('a', repositoryName, {
+      text: repo.name,
+      href: repo.html_url,
+      target: '_blank',
+    });
+
+    if (repo.description === null) {
+      createAndAppend('span', description, {
+        text: 'No Description Added Yet',
+        class: 'texts',
+      });
+    } else {
+      createAndAppend('span', description, {
+        text: repo.description,
+        class: 'texts',
+      });
+    }
+
+    createAndAppend('span', forkedNumbers, {
+      text: repo.forks,
+      class: 'texts',
+    });
+
+    createAndAppend('span', updatedTime, {
+      text: convertDataTime(repo.updated_at),
+      class: 'texts',
+    });
   }
```

Comment:
While your chosen HTML works, the information to be rendered is basically a table with two columns and four rows. Using an HTML table for this would be the most natural, intuitive representation. This is what your UI looks like on a mobile phone (perhaps also reduce the font size?):

![image](https://user-images.githubusercontent.com/2788771/65371613-c5367980-dc65-11e9-8160-ca662bc6fcbf.png)

Using an HTML table it could look like this:

![image](https://user-images.githubusercontent.com/2788771/65371621-dda69400-dc65-11e9-9382-ba3805a74ba5.png)

If this was an app in the App Store, which one would you prefer?